scripts.js

```diff
@@ -0,0 +1,23 @@
+const childProcess = require('child_process');
```

Comment:
Can you add a copyright header to this file, like the others? I guess it would be either "Copyright 2020 Mozilla Corporation" or "Copyright 2020 Mozilla Foundation", which I don't know.

Reply:
I think the MoCo one should work, but I should probably ask my manager about this. Making this a draft until I figure it out...

Reply:
Hope it's not a whole process, if so I apologize! :)

Reply:
It turns out that Copyright 2020 MoCo is indeed correct here. 👍

Reply:
Thanks for sorting that out!